Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -838,12 +838,24 @@ def licenses(family_directory):
 
 @register_condition
 @condition
-def license(licenses):
+def license_path(licenses):
   """Get license path"""
   # return license if there is exactly one license
   return licenses[0] if len(licenses) == 1 else None
 
 
+@register_condition
+@condition
+def license(license_path):
+  """Get license name"""
+  if "LICENSE.txt" in license_path:
```

Comment:
I don't think this is the right check. What you could do is:
```py
if license_path.endswith('/LICENSE.txt'):
```

 to be more precise. Murphy's law dictates that someone will eventually have a `license` like:  `./LICENSE.txt/font/OFL.txt`!

Though, Im still in favor of:

```py
return os.basename(license_path)
```
Also, because the `licenses` getter already knows what valid licenses are.